public/app/themes/clarity/functions.php

```diff
@@ -120,6 +119,9 @@
 new MOJ\Intranet\WPOffloadMedia();
 new MOJ\Intranet\WPElasticPress();
 
+$search = new MOJ\Intranet\Search();
+$search->hooks();
```

Comment:
Do we have to notate it like this? Can we follow the pattern of include and or do we need other $search variables in the `functions.php` script?